File: Pratica8/python/serial.py
filename = "C:/Users/20221en20108/Documents/LingProgGit/Pratica8/test_file.txt"

try:
    with open(filename, 'r') as file:
        for line in file:
            print(line.rstrip('\n'))
except IOError:
    print("Erro ao abrir o arquivo.")

File: Pratica8/matlab/serial.m
filename = 'C:/Users/20221en20108/Documents/LingProgGit/Pratica8/test_file.txt';

fid = fopen(filename, 'r');
if fid == -1
    error('Erro ao abrir o arquivo.');
end

while ~feof(fid)
    line = fgetl(fid); % lê uma linha do arquivo
    disp(line)         % exibe a linha no console
end

fclose(fid);
